webui/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I think we shouldn't specify the lang attribute because the application is multilingual